hubcap/setup.py

```diff
@@ -0,0 +1,58 @@
+import collections
+import datetime
+import json
+import logging
+import os
+import re
+import requests
+
+from pathlib import Path
+
+NOW = int(datetime.datetime.now().timestamp())
+NOW_ISO = datetime.datetime.now(datetime.timezone.utc).astimezone().isoformat()
+
+logging.basicConfig(
+    format='%(asctime)s %(levelname)-8s %(message)s',
+    level=logging.INFO,
+    datefmt='%Y-%m-%d %H:%M:%S'
+)
+
+PKG_EXCLUSION_OVERRIDES = {
+    'dbt-labs': ['bing-ads', 'recurly', 'shopify', 'purecloud', 'zendesk', 'stripe', 'outbrain', 'taboola',
+                 'quickbooks', 'bing-ads', 'taboola', 'heap', 'ecommerce', 'facebook-ads', 'zendesk', 'stripe',
+                 'purecloud'],
+    'beautypie': ['dbt-google-analytics-360', 'dbt-zendesk-support']
+}
```

Comment:
Could we make the exclude list another `.json` config file?

Reply:
We can but I'd rather just exclude these at the source frankly. This was a patch to get things work. I'd prefer to just block people from adding their packages that don't meet specifications rather than doing an exclusion dance. Is there a compelling reason that I may be missing for why we must keep everything in hub.json?

Reply:
Did the thing and left a comment to remind we should just change the policy here probably.